core/Enclave/key_factory.cpp

```diff
@@ -282,6 +293,11 @@ sgx_status_t ehsm_create_rsa_key(ehsm_keyblob_t *cmk)
         goto out;
     }
 
+    if (!test_RSA_keypair(rsa_keypair))
+    {
+        goto out;
+    }
```

Comment:
remove this.
self test only exists in core/Enclave/enclave_self_test.cpp

Reply:
it used as the pari-wise test, not for self test, so won't need to move to the enclave_self_test.cpp.

and how about the other asymmetric keypair? just see the rsa

Reply:
why need it? every time ehsm_create_rsa_key is called, this test should be executed.